[infra] 初回マイグレーション時、DBへの接続に失敗することがある

```
$ docker compose run --rm db-migration bash -c 'make migrate-up db_url=${DB_URL}'
migrate -database postgres://postgres:password@db:5432/twinte_db?sslmode=disable -path ./migrations up
error: failed to open database: dial tcp 192.168.97.2:5432: connect: connection refused
make: *** [Makefile:7: migrate-up] Error 1
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -83,7 +83,12 @@ services:
     ports:
       - 5432:5432
     networks:
-      - twinte-network     
+      - twinte-network
+    healthcheck:
+      test: ["CMD-SHELL", 'pg_isready -U "$POSTGRES_USER"']
+      interval: 5s
+      timeout: 5s
+      retries: 5
 
   db-migration:
     container_name: twinte-db-migration
@@ -99,7 +104,8 @@ services:
     networks:
       - twinte-network
     depends_on:
-      - db
+      db:
+        condition: service_healthy
 
   parser:
     container_name: twinte-parser
@@ -121,7 +127,7 @@ services:
     volumes:
       - .:/usr/src/twin-te
     networks:
-      - twinte-network      
+      - twinte-network
 
 networks:
   twinte-network:

